File: sdk/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineResponse400.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param message 
 * @param status 
 */
data class InlineResponse400 (

        @JsonProperty("message") val message: String? = null,

        @JsonProperty("status") val status: Int? = null
) {

}

File: sdk/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineResponse400.scala

package org.openapitools.client.model


case class InlineResponse400 (
    _message: Option[String],
    _status: Option[Integer]
)
object InlineResponse400 {
    def toStringBody(var_message: Object, var_status: Object) =
        s"""
        | {
        | "message":$var_message,"status":$var_status
        | }
        """.stripMargin
}
